test/compiler_spec.js

```diff
@@ -29,8 +29,9 @@ describe('Vue component compiler', function () {
   test('empty')
   test('less')
   test('sass')
-  test('myth')
+  // test('myth')
```

Comment:
I figured out that it's due to myth's dependencies has been updated, transition does not need to be prefixed anymore. https://github.com/davidhund/shouldiprefix/commit/7f6aa0c8021e490bd381b8120e977264e90aa117

I will update the test case to make it pass.